app/queries/events_query.rb

```diff
@@ -5,9 +5,9 @@ def call
     events = organization.clickhouse_events_store? ? Clickhouse::EventsRaw : Event
     events = events.where(organization_id: organization.id)
     events = paginate(events)
-
     events = events.order(timestamp: :desc) unless organization.clickhouse_events_store?
     events = events.order(ingested_at: :desc) if organization.clickhouse_events_store?
+    events = apply_consistent_ordering(events)
```

Comment:
In that case, we are already ordering the events in the previous lines (depending on the event store configured at organization level), we should not reapply any other ordering as no index is covering this correctly and those one have a huge number of rows

Also, if the event store is clickhouse, the table does not have an ID column.
It shows that a test case covering this is missing :D